elastic/kibana
# [Dev Tools] Change breadcrumbs and docTitle when toggling between apps

Fixes https://github.com/elastic/kibana/issues/117307

### Summary

This PR adds the missing breadcrumb belonging to an app that is opened in the dev tools UI and also keeps the doc title in sync.

**How to test**
1. Start elasticsearch with `yarn es snapshot --license=trial` 
2. Start kibana with `yarn start`
3. Navigate to `Dev Tools`
4. Verify that toggling between apps correctly changes the breadcrumbs and the doc title


**Screenshots**

<details>
<summary>Overview of changes</summary>

// Breadcrumbs + doc title 
<img width="727" alt="Screenshot 2021-11-17 at 15 03 00" src="https://user-images.githubusercontent.com/1191206/142215047-06ca5f38-174b-45c0-a731-d23103855f27.png">


</details>
